packages/engine/src/scene/functions/loadGLTFModel.ts

```diff
@@ -124,7 +124,7 @@ export const parseObjectComponentsFromGLTF = (
 
   if (!scene) return {}
 
-  scene.traverse((mesh: Mesh) => {
+  iterateObject3D(scene, (mesh: Mesh) => {
```

Comment:
let's revert this to scene.traverse, as this is specific setup logic that turns the threejs hierarchy into ECS hierarchy